app/models/user_event.rb

```diff
@@ -13,7 +13,7 @@ class UserEvent < ApplicationRecord
   scope :participants, -> { where(event_role: "participant") }
   scope :co_hosts, -> { where(event_role: "co_host") }
   scope :for_user, ->(user_id) { where(user_id: user_id) }
-  scope :order_by_priority, -> { order("priority DESC NULLS LAST") }
```

Comment:
@snehithlal This is intended to eliminate any kind of raw sql, and instead use arel for the queries which we cant otherwise write using AR. Arel is considered private API but we can write enough tests to ensure it wont break between version releases.

Reply:
Ok. Then u can merge